examples/examples_go_test.go

```diff
@@ -59,6 +59,7 @@ func TestBuildCacheFromGo(t *testing.T) {
 }
 
 func TestDockerfileGo(t *testing.T) {
+	t.Skipf("Skipping test due to dockerignore shenanigans")
```

Comment:
Could have a more descriptive message here, if I saw this in code out of context I wouldn't understand why this test is skipped

Reply:
Nope, I just left litter around - this should work now, so I will unskip it.